### DISCUSSION TITLE
No change log for version 2311?

### DISCUSSION THREAD

**Discussion by HotCakeX** [microsoft/WSA#483] [General]
Where is the announcement for version 2311? It was installed about 2 days ago on my machine.

<br>

<img width="575" alt="image" src="https://github.com/microsoft/WSA/assets/118815227/e1127eb1-d17f-41e8-be1a-80fbc135f66d">

**Comment by Mitia99** [comment]
@jaholme

**Reply by jaholme** [reply]
#489
